test/openssl/test_ssl.rb

```diff
@@ -804,6 +804,71 @@ def socketpair
     end
   end
 
+  def test_keylog_cb
+    pend "Keylog callback is not supported" if !openssl?(1, 1, 1) || libressl?
+
+    possible_versions = [
+      OpenSSL::SSL::SSL3_VERSION,
+      OpenSSL::SSL::TLS1_VERSION,
+      OpenSSL::SSL::TLS1_1_VERSION,
+      OpenSSL::SSL::TLS1_2_VERSION
+    ]
+
+    possible_versions.each do |ver|
+      prefix = 'CLIENT_RANDOM'
+      context = OpenSSL::SSL::SSLContext.new
+      context.min_version = context.max_version = ver
+
+      cb_called = false
+      context.keylog_cb = proc do |ary|
+        cb_called = true
+        _sock, line = ary
+        assert_equal(line.split.first, prefix)
```

Comment:
`assert_equal(expected, actual)`

Reply:
Fixed in [9f4fa70](https://github.com/ruby/openssl/pull/536/commits/9f4fa7081876ff3f6dad140da51e9f3b6240bc20)